metadata-db-app/src/cmr/metadata_db/services/messages.clj

```diff
@@ -215,6 +215,11 @@
                "Variable associations are system level entities.")
           provider-id))
 
+(defn service-associations-only-system-level [provider-id]
```

Comment:
Nitpick: put argument on its own line

Reply:
Looks like almost all the functions in this namespace are using this style of declaration, perhaps for the ease of readability. I am not a fan of this type of messaging functions in general and don't want my finger print to be all over this namespace. So I am only changing a couple of the functions. :)